tiledb/sm/tile/tile.h

```diff
@@ -50,6 +51,61 @@ namespace sm {
  * offset in the file where the tile data begin, plus the tile size.
  */
 class Tile {
+ public:
+  /**
+   * Computes the chunk size for a tile.
+   *
+   * @param tile_size The total size of the tile.
+   * @param tile_dim_num The number of coordinate dimensions.
+   * @param tile_cell_size The cell size.
+   * @param chunk_size Mutates to the calculated chunk size.
+   * @return Status
+   */
+  static Status compute_chunk_size(
```

Comment:
Could you please elaborate on the rationale behind this function? In my mind, the chunk size is always fixed and should reflect a typical size of L1 cache.